src/commands/synthetics/__tests__/run-tests-lib.test.ts

```diff
@@ -767,12 +767,11 @@ describe('run-test', () => {
         )
       ).resolves.toEqual([
         {
-          testOverrides: {deviceIds: ['chrome.laptop_large'], startUrl, locations: ['aws:ap-northeast-1']},
+          testOverrides: {startUrl, locations},
           id: 'abc-def-ghi',
-          suite: 'Suite 1',
```

Comment:
Why are we removing the suite and the deviceIds?

Reply:
Because we aren't reading test configuration files anymore when `files` is not passed explicitly. So in this unit test, we lose any information coming from `fakeSuites`.